src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Controllers/HelloWorldController.cs

```diff
@@ -1,22 +1,75 @@
+using Asp.Versioning;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
 using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
 
 namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Controllers;
 
+[ApiVersion(1.0)]
+[ApiVersion(2.0)]
 [ApiController]
-[Route("/HelloWorld")]
+[Route("api/v{version:apiVersion}/HelloWorld")]
 public class HelloWorldController(PublicDataDbContext publicDataDbContext) : ControllerBase
 {
-    [HttpGet]
-    public async Task<ActionResult<string>> Get()
+    /// <summary>
+    /// List v1
+    /// </summary>
+    /// <remarks>
+    /// Test the API can list with `GET` in v1.
+    /// </remarks>
+    [HttpGet(Name = "List")]
```

Comment:
Could we use `nameof` here? (same for other actions)

Reply:
This was just for illustrating how to set the endpoint's documented operation ID. However, I've changed this behaviour so that it:

1. Tries to use route name defined by `Http*` or `Route` attribute (i.e. the above)
2. If no route name, fallback to the method name itself

This means that in most cases, the method name by itself will suffice (no need for the name in an attribute)